src/components/Icon/components/IconActionEmptyCircle.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const IconActionCircle = (props: React.SVGAttributes<SVGSVGElement>) => (
```

Comment:
`props` isn't used anywhere so this can be omitted

Reply:
No worries. As it happens, I'm toying with removing the props spreading from all icons as everything they really need is governed by the parent Icon component (height, width and fill colour). The spread is convenient but also opens it up to bugs.